UDPBroadcast/UDPBroadcastConnection.swift

```diff
@@ -49,6 +49,7 @@ open class UDPBroadcastConnection {
         )
         
         self.handler = handler
+        _ = createSocket(bindIt: true)
```

Comment:
We wouldn't want to create a socket here if we don't need it. Please add a parameter to `init()` that switches this behaviour on/off. Please change the comment above the function as well since it explicitly states that the socket will be opened only if the user sends a message.
Furthermore not all users will want to open a listening port (`bindIt == true`) here.

Reply:
👍🏻